tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -1747,7 +1747,7 @@ trait WebActionsApiBaseTests extends ControllerTestCommon with BeforeAndAfterEac
   }
 
   class TestingEntitlementProvider(config: WhiskConfig, loadBalancer: LoadBalancer)
-      extends EntitlementProvider(config, loadBalancer, InstanceId(0)) {
```

Comment:
if we can make all InstanceId essentially a StringInstanceId, then an `apply(int)` instance would provide backward compatibility so you don't have to change all the cons sites.